axelor-account/src/main/java/com/axelor/apps/account/db/repo/MoveManagementRepository.java

```diff
@@ -59,10 +60,32 @@ public Move copy(Move entity, boolean deep) {
     copy.setIgnoreInDebtRecoveryOk(false);
     copy.setPaymentVoucher(null);
     copy.setRejectOk(false);
+    copy.setInvoice(null);
+
+    List<MoveLine> moveLineList = copy.getMoveLineList();
+
+    if (moveLineList != null) {
+      moveLineList.forEach(this::resetMoveLine);
+    }
 
     return copy;
   }
 
+  public void resetMoveLine(MoveLine moveLine) {
+    moveLine.setInvoiceReject(null);
+    moveLine.setDate(Beans.get(AppBaseService.class).getTodayDate());
+    moveLine.setExportedDirectDebitOk(false);
+    moveLine.setReimbursementStatusSelect(MoveLineRepository.REIMBURSEMENT_STATUS_NULL);
+
+    List<AnalyticMoveLine> analyticMoveLineList = moveLine.getAnalyticMoveLineList();
+
+    if (analyticMoveLineList != null) {
+      moveLine
+          .getAnalyticMoveLineList()
+          .forEach(line -> line.setDate(Beans.get(AppBaseService.class).getTodayDate()));
```

Comment:
better to get the date value from Move instead of use injection to get the today date